src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,23 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e:
+                    msg = ''
+                    for authzr in e.failed_authzrs:
+                       msg += f'\n Authorization for identifier {%s} failed.' % (
+                         authzr.body.identifier)
+                       msg += f'\n Here are the challenges that were not fulfilled:'
+                       for challenge in authzr.body.challenges:
+                         msg += \
+                               f'\n Challenge Type: {%s}' \
+                               f'\n Error information: ' \
+                               f'\n Type: {%s}' \
+                               f'\n Details: {%s} \n\n' % (
+                                   challenge.chall.typ,
+                                   challenge.error.typ if challenge.error else '',
+                                   challenge.error.detail if challenge.error else '',
+                                )
```

Comment:
@sonicaj So you want these lines removed?

```suggestion
                         msg += \
                               f'\n Challenge Type: {%s}' \
                               f'\n Error information: ' \
                               f'\n Type: {%s}' \
                               f'\n Details: {%s} \n' 
```

Reply:
Yes but what was being done after `% (` can be done directly in the f-strings like
```
msg += f'\n Challenge Type: {challenge.chall.typ}' \
             f'\n Error information: ' \
             f'\n Type: {challenge.error.typ if challenge.error else ""}' \
             f'\n Details: {challenge.error.detail if challenge.error else ""} \n\n'
```

Reply:
Would you mind next time using Github suggestions? It's rather... tedieus to have to copy paste feedback manually every time.

Reply:
```suggestion
                         msg += \
                               f'\n Challenge Type: {challenge.chall.typ}' \
                               f'\n Error information: ' \
                               f'\n Type: {challenge.error.typ if challenge.error else ""}' \
                               f'\n Details: {challenge.error.detail if challenge.error else ""} \n\n'
```